doc/chrome.md

```diff
@@ -17,6 +17,21 @@ reference of the available startup options.
 (def foreground-chrome (chrome/launch {:headless false}))
 ```
 
+### Old and new headless
+
+Since version 96, Chrome has a new headless mode that allows users to get the
+full browser functionality (even run extensions). Between versions 96 to 108 it was
+--headless=chrome, after version 109 --headless=new. `cuic` supports both old and new
+headless modes.
+
+```clojure
+;; launch in old headless mode
+(def old-headless-chrome (chrome/launch {:headless "old"}))
+
+;; launch in new headless mode
+(def old-headless-chrome (chrome/launch {:headless "new"}))
```

Comment:
Can we also add a note about previous boolean usage and perhaps add a deprecation notice?

```clj
;; deprecated: equivalent with {:headless "old"} but may be removed in future
(def old-headless-chrome (chrome/launch {:headless true}))
```

Reply:
Will do!